README.md

```diff
@@ -130,19 +130,19 @@ $ skopeo inspect docker://registry.fedoraproject.org/fedora:latest | jq '.Digest
 ## Copying images
 
 `skopeo` can copy container images between various storage mechanisms, including:
-* Container registries
+* **Container registries**
```

Comment:
Is there a semantic point to this? it’s just a list.

Reply:
The main logic is to have a blog or website look better with perfect spacings and maintaining a good header-text ratio, which is what I tried. Is it good??

Reply:
I mean, we are not in control of the CSS on this site anyway (and most of us probably wouldn’t _want_ to tinker with it if we could); so the way this content gets longevity and resilience to (GitHub’s and others’) CSS changes is by focusing on the semantics.

Reply:
Okay!!! I got the point. My mistake